server/bin/pbench-trampoline

```diff
@@ -6,14 +6,14 @@ _basename=$(basename ${0})
 if [[ ! ":${PATH}:" =~ ":${_dirname}:" ]]; then
    PATH=${_dirname}${PATH:+:}${PATH}; export PATH
 fi
-if command -v python3 > /dev/null 2>&1 ;then
+if command -v python3.9 > /dev/null 2>&1 ;then
     _pythoncmd=""
 else
-    printf -- "${_basename}: pbench-trampoline needs python3\n" >&2
+    printf -- "${_basename}: pbench-trampoline needs python3.9\n" >&2
     exit 1
 fi
 _installdir=$(dirname ${_dirname})
-export PYTHONPATH=${_installdir}/lib:${_installdir}/lib/python3.8/site-packages:${PYTHONPATH}
+export PYTHONPATH=${_installdir}/lib:${_installdir}/lib/python3.9/site-packages:${_installdir}/lib64/python3.9/site-packages:${PYTHONPATH}
```

Comment:
_[Continuing from my [previous comment](https://github.com/distributed-system-analysis/pbench/pull/3062#discussion_r1019410737) with more context.]_

I grant that PR is a change for 0.69, so perhaps there's no point in messing with `_pythoncmd`.  

I grant that the change at line 16 is key, and I expect it is that which actually addresses the problem.

My concern is with the test at line 9.  First, I don't think that that test has any real effect -- if things are "OK", then we end up ignoring `_pythoncmd`, and all that matters is that whatever the shebangs reference exists and is compatible with the `PYTHONPATH` which we set up at line 16.  The problem is that that test _should_ be checking for whatever the shebangs are going to reference, but, to the extent that it actually does so, it's only by coincidence (and I can't think of a "good" way for it to do the proper check).

So, I would propose simply deleting lines 9-14:  if the necessary Python is missing, we'll find out when the Python script is invoked and the shebang reference cannot be satisfied.  (And, if the necessary Python is _present_, then these lines don't actually have any effect, assuming that they are actually testing for the right Python, so deleting them does no harm.)

Reply:
Certainly your comments make sense.  I don't think it is worth making any further changes since this is what is working right now in production.

Reply:
I don't know, good question ... perhaps worth tracking in a separate GitHub issue ... ;)

Reply:
Not if Click will make the trampolines go away....